packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card-section.tsx

```diff
@@ -0,0 +1,80 @@
+import { Text } from '@deriv/components';
+import classNames from 'classnames';
+import React, { SyntheticEvent } from 'react';
+import { TClickableDescription, TJurisdictionCardSectionTitleIndicators } from 'Components/props.types';
+import { TJurisdictionCardSectionProps } from '../props.types';
+
+const JurisdictionCardSection = ({ cardSectionItem, flipCard }: TJurisdictionCardSectionProps) => {
+    const handleCardFlip = (event: SyntheticEvent) => {
+        event.stopPropagation();
+        flipCard();
+    };
+
+    const renderTitleIndicator = (titleIndicators: TJurisdictionCardSectionTitleIndicators) => {
+        switch (titleIndicators.type) {
+            case 'displayText':
+                return (
+                    <div
+                        className={classNames(
+                            'cfd-card-section-title-indicator',
+                            `cfd-card-section-title-indicator__${titleIndicators.displayTextSkinColor}`
+                        )}
```

Comment:
do we need these 2 class?

Reply:
@amina-deriv one class is for required styling and the other is for giving the background color as per design. shall i refactor it in some other way? (this file is outdated, please see the latest file content)